substrate/frame/staking/src/lib.rs

```diff
@@ -1127,10 +1131,64 @@ impl<T: Config> Convert<T::AccountId, Option<Exposure<T::AccountId, BalanceOf<T>
 	}
 }
 
-pub struct NullIdentity;
-impl<T> Convert<T, Option<()>> for NullIdentity {
-	fn convert(_: T) -> Option<()> {
-		Some(())
+/// A marker type representing the presence of a validator. Encodes as a unit type.
+pub type Existence = ();
```

Comment:
fwiw this is a type _alias_ to `()`, what we usually call a marker type is actually a different type, such as `pub struct Existence`

Reply:
Subjective but reading the code a bit, I would have found `NullOrLegacyExposure` a better name.